libnavigation-core/src/main/java/com/mapbox/navigation/core/history/MapboxHistoryReader.kt

```diff
@@ -0,0 +1,46 @@
+package com.mapbox.navigation.core.history
+
+import com.mapbox.navigation.core.history.model.HistoryEvent
+import com.mapbox.navigation.core.history.model.HistoryEventMapper
+import com.mapbox.navigator.HistoryReader
+import java.io.Closeable
+
+/**
+ * Allows you to read history files previously saved by [MapboxHistoryRecorder].
+ * All files in the [MapboxHistoryRecorder.fileDirectory] can be read with this reader.
+ *
+ * @param filePath absolute path to a file containing the native history file.
+ */
+class MapboxHistoryReader(
```

Comment:
This isn't currently mentioned in any other docs, developers will have a hard time finding this class. Let's mention it in recorder interfaces.

Reply:
Good call! Added